tests/api/client/test_local_client.py

```diff
@@ -76,6 +76,7 @@ def test_trigger_dag(self, mock):
                 conf=None,
                 external_trigger=True,
                 dag_hash=ANY,
+                data_interval=ANY,
```

Comment:
This should be checkable, can you try to not use `ANY` here (and the one below)?

Reply:
Sure. I also updated the test to check the value of "dag_hash".